pkg/controller/control.go

```diff
@@ -1220,9 +1220,9 @@ func (c *Controller) handleErrorNoLock(err error) error {
 				}
 			}
 
-			if err := c.handleDiskNoSpaceErrorForReplicas(noSpaceErrMap); err != nil {
-				log.WithError(err).Error("All disks of writable replicas are out of space")
-				return err
+			if noSpaceErr := c.handleDiskNoSpaceErrorForReplicas(noSpaceErrMap); noSpaceErr != nil {
+				log.WithError(noSpaceErr).Error("All disks of writable replicas are out of space")
```

Comment:
Will it lead to flooding messages?

Reply:
Yes, and we agreed to modify it ot a `Debug` log.
Modified. Thanks.